src/styles/button.scss

```diff
@@ -0,0 +1,38 @@
+.button {
```

Comment:
All clickable objects should be highlighted to clearly indicate the user they're interactable [Reference](https://mate-academy.github.io/layout_task-guideline/html-css-code-style-rules#:~:text=Hightlight%20clickable%20element%20with%20cursor%3A%20pointer%20and%20some%20%3Ahover%20styles)